src/java_tools/junitrunner/java/com/google/testing/coverage/ClassProbesMapper.java

```diff
@@ -52,7 +52,7 @@ public Map<Integer, BranchExp> result() {
   public ClassProbesMapper(String className) {
     classLineToBranchExp = new TreeMap<Integer, BranchExp>();
     stringPool = new StringPool();
-    className = stringPool.get(className);
+    this.className = stringPool.get(className);
```

Comment:
I have a separate change out for review for this which should be submitted soon.

Reply:
You can leave it in for now. The fix should show up in the next hour, and if you want you could rebase at that point. But if not, the merge should be resolved when the PR is imported.

Reply:
I've rebased on master and the diff no longer appears for `ClassProbesMapper`